src/python/ensembl/io/genomio/genbank/extract_data.py

```diff
@@ -455,8 +459,8 @@ def _write_genome_json(self):
         }
 
         if not genome_data["species"]["production_name"]:
-            print(
-                f"Warning: please add the relevant production_name for this genome in {self.files['genome']}"
+            logging.warning(
+                f"Please add the relevant production_name for this genome in {self.files['genome']}"
```

Comment:
```suggestion
                f"Please add the relevant production name for this genome in {self.files['genome']}"
```
Just to follow the same format as previous messages (unless this is a slightly different case).

Reply:
I would keep production_name because it's a specific term that we use (not just a name for production).